### PR TITLE
time-chart

### PR DESCRIPTION
**Pull request by kghamilton89**
## Time Chart Settings

* Primary a **format-only** refactor of Time Chart. (EDIT: see final comment)
  * I began refactoring content before our new strategy began, so some of those changes remain here.

### CODE REVIEW

**Comment by kghamilton89** [comment]
@raipc O Benevolent Time Lord, the linter is calling working links dead. They are a bit strange (contain `/` character, but work).

**Comment by kghamilton89** [comment]
This PR now contains complete content, chart, and formatting refactor.

**Comment by kghamilton89** [comment]
Ran all example configurations through the plugin, a few minor changes were needed. Most were issues with the plugin itself.

**Comment by kghamilton89** [comment]
Reviewed all settings against original documentation. Several have been moved to Shared Settings, but Time Chart-specific are all included.

File: widgets/time-chart/README.md [hunk 1]
```diff
@@ -1,79 +1,90 @@
...
+<a name="forecast-style"></a>[`forecast-style`](#forecast-style) | `forecast-style = stroke: magenta; opacity: 0.25` | CSS style applied to [forecasts](https://axibase.com/docs/atsd/forecasting/) in `column` and `column-stack` mode.| [↗](https://apps.axibase.com/chartlab/b3e87541)
+
+## Series Settings
+
+* The settings apply to the `[series]` section.
+* [Shared](../shared/README.md#series-settings) `[series]` settings are inherited.
+
+Name | Example | Description | &nbsp;
+:--|:--|:--|:--
+<a name="display"></a>[`display`](#display) | `display = value >= top(3)` | Display expression.<br>Refer to [Expression Syntax Documentation](https://axibase.com/docs/atsd/administration/metric-persistence-filter.html#expression-syntax) for more information.| [↗](https://apps.axibase.com/chartlab/9371863f)
```
Comment (rodionos) [inline widgets/time-chart/README.md]: Should `display` and `enabled` settings be part of Shared doc?

File: widgets/time-chart/README.md [hunk 2]
```diff
@@ -1,79 +1,90 @@
...
+<a name="last-marker"></a>[`last-marker`](#last-marker) | `last-marker = false` | Show last series value marker.<br>Possible values: `false`, `true`. Default value: `true`. | [↗](https://apps.axibase.com/chartlab/5a54b7ce)
+<a name="horizontal-grid"></a>[`horizontal-grid`](#horizontal-grid) | `horizontal-grid = false` | Show horizontal grid.<br>Possible values: `false`, `true`.<br>Default value: `true`. | [↗](https://apps.axibase.com/chartlab/a28cbb11)
+<a name="vertical-grid"></a>[`vertical-grid`](#vertical-grid) | `vertical-grid = false` | Show vertical grid.<br>Possible values: `false`, `true`.<br>Default value: `true`. | [↗](https://apps.axibase.com/chartlab/50895463)
+<a name="pointer-position"></a>[`pointer-position`](#pointer-position) | `pointer-position = left` | Location of final value pointer.<br>Possible values: `left`, `right`, `none`.<br>Default value: `right`.| [↗](https://apps.axibase.com/chartlab/33aa5216)
+<a name="auto-scale"></a>[`auto-scale`](#auto-scale) | `auto-scale = true` | Scale the graph based on series values.<br>Possible values: `false`, `true`.<br>Default value: `false`. | [↗](https://apps.axibase.com/chartlab/4256fd76)
```
Comment (rodionos) [inline widgets/time-chart/README.md]: Value axis and not graph

File: widgets/time-chart/README.md [hunk 3]
```diff
@@ -111,26 +122,24 @@ Modify the `60` component of the setting to change the number of minutes to sear
...
-To remove selection intervals and aggregation periods, drag and drop the interval or period off the Time Chart controls. In the example below, `1w` is removed.
-
-![](./images/removed-timespan.png)
+To remove selection intervals and aggregation periods, drag and drop the interval or period off the **Time Chart** controls. In the example below, `1w` is removed.
 
-Modify time chart panel controls with the `defaultChartConfig.script` setting at the `[configuration]` level.
+Modify **Time Chart** panel controls with the `defaultChartConfig.script` setting at the `[configuration]` level.
 
 ```ls
 defaultChartConfig.script = widget.chart.panels[1].expand(8)
```
Comment (rodionos) [inline widgets/time-chart/README.md]: This hack is no longer necessary: defaultChartConfig.script = widget.chart.panels[1].expand(8)
We already have two settings to control the panels.

File: widgets/time-chart/README.md [hunk 4]
```diff
@@ -1,79 +1,90 @@
...
+    entity = nurswgvml007
+    metric = cpu_busy
+```
 
-### Series Visibility
+![](./images/time-chart.png)
+
+[![](../../images/button.png)](https://apps.axibase.com/chartlab/3146896d)
+
+## Widget Settings
```
Comment (rodionos) [inline widgets/time-chart/README.md]: Lets move some settings into subsections:

* Time axis (rotate-ticks, centralize-ticks, etc.)
* Value axis (auto-scale, etc)

Three tables in total.

File: widgets/time-chart/README.md (hunk 1 above)
Comment (kghamilton89) [reply widgets/time-chart/README.md]: It looks like they should. They work with all widgets. I'll remove them and put them in the shared settings docs.
EDIT: They are already contained there.

File: widgets/time-chart/README.md (hunk 2 above)
Comment (kghamilton89) [reply widgets/time-chart/README.md]: Changed.

File: widgets/time-chart/README.md (hunk 3 above)
Comment (kghamilton89) [reply widgets/time-chart/README.md]: I'll remove that section.

File: widgets/time-chart/README.md (hunk 4 above)
Comment (kghamilton89) [reply widgets/time-chart/README.md]: Three sub-tables are in the document now

**Comment by kghamilton89** [comment]
* `expand-panels` setting updated to include possible values: `false`, `true`, `compact`. Example chart updated to show these options.

**Pull request merged**